sycl/test/basic_tests/image_accessor_range.cpp

```diff
@@ -0,0 +1,117 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
+// RUN: %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER

```

Comment:
Please, move the test to  `llvm-test-suite` repo since it has dependencies on low-level runtimes.